cassovary-core/src/main/scala/com/twitter/cassovary/graph/SortedNeighborsNodeOps.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2014 Twitter, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.twitter.cassovary.graph
+
+import com.twitter.cassovary.util.SortedArrayOps
+
+trait SortedNeighborsNodeOps {
+  self: Node =>
+
+  override protected def containsNode(nodeIds: Seq[Int], queryNodeId: Int): Boolean = {
```

Comment:
If you don't do an 'override', then the compiler will choose this implementation based on the method signature which could be an array here. i.e.,
protected def containsNode(nodeIds: Array[Int], queryNodeId: Int): Boolean = ....

then the compiler will choose this and you don't have to write the code for or do an .toArray at runtime (I think)